_includes/page-footer.html

```diff
@@ -1,20 +1,21 @@
 {%- if include.pagination -%}
-  {%- assign footer_content = 'Pagination' -%}
-  {%- assign class = 'pagination' %}
+{%- assign footer_content = 'Pagination' -%}
+{%- assign class = 'pagination' %}
 {%- elsif page.layout == 'post' -%}
-  {% assign view_all_url = page.view_all_url %}
-  
-  {% if page.collection == 'briefs' %}
-    {% assign view_all_url = page.view_all_url | append: page.title %}
-  {% endif %}
+{% assign view_all_url = page.view_all_url %}
 
-  {%- capture footer_content -%}<a href="{{ view_all_url | relative_url }}" class="page__footer-link"><i class="icon-docs"></i> {{
+{% if page.collection == 'briefs' %}
```

Comment:
This if statement can be removed. As `page.layout == 'post'` will be true for briefs, it's already covered under the previous if statement.